overlays/cinder-volume.yaml

```diff
@@ -0,0 +1,33 @@
+# Variables
+debug:                      &debug                     True
+verbose:                    &verbose                   True
+openstack_origin:           &openstack_origin          __OS_ORIGIN__
+source:                     &source                    __SOURCE__
+
+ssl_ca:                     &ssl_ca                    __SSL_CA__
+ssl_cert:                   &ssl_cert                  __SSL_CERT__
+ssl_key:                    &ssl_key                   __SSL_KEY__
+
+applications:
+  cinder:
+    options:
+      enabled-services: api,scheduler
+  cinder-volume:
+    num_units: __NUM_CINDER_UNITS__
+    constraints: mem=2G
+    charm: __CHARM_STORE____CHARM_CS_NS____CHARM_CH_PREFIX__cinder
+    options:
+      debug: *debug
+      verbose: *verbose
+      enabled-services: volume
+      block-device:
+      glance-api-version: 2
+      openstack-origin: *openstack_origin
+      ssl_ca: *ssl_ca
+      ssl_cert: *ssl_cert
+      ssl_key: *ssl_key
+relations:
+  - [ cinder-volume:shared-db, __MYSQL_INTERFACE__ ]
+  - [ cinder-volume, rabbitmq-server ]
+# Add when https://review.opendev.org/c/openstack/charm-cinder/+/845558 is merged
```

Comment:
the mentioned patch is now merged so i guess you need to uncomment this?

Reply:
I might hold off on merging that until the patch that makes that works is backported all the way to ussuri at least. Or otherwise we have to try add some kind of conditional currently it only works in Yoga+ and only on edge not stable right now I think.

Thoughts?